cost-analyzer/values.yaml

```
@@ -71,6 +71,8 @@ global:
     basicAuth:
       # username: user
       # password: pwd
+    # imagePullSecrets:
+      # - name: "image-pull-secret"
```

```suggestion
    imagePullSecrets: []
      # - name: "image-pull-secret"
```

Minor thing. When adding a new config to the values.yaml we try to define a default value for it. In this case it should be an empty array. Then there's a comment below providing an example.

https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/DEVELOPMENT.md#yaml-style-guidelines

If urgent, ok to ignore this and go ahead with merge.

---

Not urgent at all. I appreciate the comment, @thomasvn :)